atk/src/auto/object.rs

```diff
@@ -814,19 +821,27 @@ impl<O: IsA<Object>> AtkObjectExt for O {
         }
         unsafe {
             let f: Box_<F> = Box_::new(f);
-            connect_raw(
+            let detailed_signal_name =
+                detail.map(|name| format!("active-descendant-changed::{}\0", name));
+            let signal_name: &[u8] = detailed_signal_name
```

Comment:
I don't recall the explicit type being needed here, both `&b"..."[..]` and `.as_bytes()` return `&[u8]`.

Reply:
It's more documentation for anyone looking at the generated code than anything else.

Reply:
It's especially useful because we need a blanket cast below because of `char*` in C having different signedness depending on the platform.

Reply:
Yup, bytes are always `u8` yet `c_char` has a different definition based on the platform. Only way to get that consistent is to use a `CStr` iirc, which is another unnecessary can of worms here.